### PR TITLE
docs: add Single Sing-On document

### PR DESCRIPTION
**Pull request by leo220yuyaodog**
close: #344

### CODE REVIEW

**Comment by hsluoyz** [comment]
@Selflocking

**Review by Selflocking — CHANGES_REQUESTED** [review]
image
Reason: https://github.com/casbin/casbin-website-v2/pull/41

File: docs/how-to-connect/single-sign-on.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+We offer this single sign-on, you just need to:
+- Enable Auto signin button.
+- Fill in the URL for home page.
+- Add a **Silent Signin** function to the application home page.
+
+:::note
+The basic sign method provided by Casdoor allows users to log in to other applications in the organization by 
+selecting the user who is currently logged in or using another account. 
+
+After enable the auto singin, the selection box will not display, the logged user will log in directly.
```
Comment (Selflocking) [inline docs/how-to-connect/single-sign-on.md:21]: `singin`, typo

**Review by Selflocking — CHANGES_REQUESTED** [review]

File: docs/how-to-connect/single-sign-on.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+You have connected Casdoor and configured more than one application in an organization. You want users to sign in once 
+to any app in the organization, and then be able to sign in when they go to another app, without any extra clicks. 
+
+We offer this single sign-on, you just need to:
+- Enable Auto signin button.
+- Fill in the URL for home page.
+- Add a **Silent Signin** function to the application home page.
+
+:::note
+The basic sign method provided by Casdoor allows users to log in to other applications in the organization by
```
Comment (Selflocking) [inline docs/how-to-connect/single-sign-on.md:18]:
```suggestion
The basic signin method provided by Casdoor allows users to log in to other applications in the organization by 
```
is it a typo?

File: docs/how-to-connect/single-sign-on.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+![sso_home.png](/img/connect/sso_home.png)
+2. Enable Auto signin button.
+
+![sso_signin.png](/img/connect/sso_signin.png)
+
+## Add Silent Signin
+
+In fact, we implement auto login by carrying parameters on the URL. So your applications need to have a method to trigger
+the login after jumping to the URL. We support [casdoor-react-sdk](https://github.com/casdoor/casdoor-react-sdk) for you
+to quickly implement the feature.  You can see details in [use-in-react](https://github.com/casdoor/casdoor-react-sdk#use-in-react).
```
Comment (Selflocking) [inline docs/how-to-connect/single-sign-on.md:37]:
```suggestion
to quickly implement the feature. You can see details in [use-in-react](https://github.com/casdoor/casdoor-react-sdk#use-in-react).
```

**Review by Selflocking — CHANGES_REQUESTED** [review]

File: docs/how-to-connect/single-sign-on.md (hunk 2 above)
Comment (leo220yuyaodog) [reply docs/how-to-connect/single-sign-on.md]: Thanks, it has been corrected.

**Review by Selflocking — APPROVED** [review]

**Pull request merged**